docs/source/index.rst

```diff
@@ -19,12 +19,11 @@ ensuring compliance to security policies, tag policies, garbage
 collection of unused resources, and cost management from a single
 tool. 
 
-YAML Policies are the core concept of Cloud Custodian.  They take actions 
-on resources specified by filters. Custodian adheres to a compliance
-as code principle, so you can validate, dry-run, and code review on
-changes to your policies.
+The core concept is a stateless rule engine, that filters and takes actions on resources.
```

Comment:
This stateless rules engine bit is already in the paragraphs above (line 11).    I'd just drop the sentence and start this with "Cloud Custodian can be bound to serverless event streams across multiple ..."